frontend/app/src/components/status/notifications/NotificationPopup.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-snackbar
+    v-model="notification.display"
+    class="notification-popup"
+    :timeout="notification.duration"
+    light
+    top
+    right
+    app
+    rounded
+    width="400px"
+    @input="displayed(notification.id)"
+  >
+    <notification
+      :notification="notification"
+      @dismiss="dismiss(notification.id)"
+    />
+    <v-tooltip v-if="queue.length > 0" bottom>
+      <template #activator="{ on }">
+        <div class="notification-popup__count__wrapper" v-on="on">
+          <v-badge
+            inline
+            :content="queue.length"
+            color="info"
+            class="notification-popup__count"
+          />
+        </div>
+      </template>
+      <span>Number of Pending notifications</span>
+    </v-tooltip>
+  </v-snackbar>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator';
```

Comment:
I don't know where but I guess somewhere in this file. Can you please make it that if the user clicks anywhere else the notifications sidebar goes away. It's really annoying to have to figure out that one has to press the top left `<` in order to have it move out of the way.

With notifications I would expect clicking anywhere else they would go away. And by clicking anywhere else I mean even at unclickable elements. Just any click somewhere else.

![Peek 2020-08-18 13-31](https://user-images.githubusercontent.com/1658405/90508691-74baeb00-e158-11ea-92ed-6a44cfb32187.gif)

Reply:
Sure that should be relatively easy. It alredy works on the smaller screen sizes.